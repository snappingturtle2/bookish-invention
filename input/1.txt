2
HEX_STRING
abcd
HEX_STRING
abcd
DUMP
SHL
1 0 4
DUMP
SHL
1 0 16
DUMP
END