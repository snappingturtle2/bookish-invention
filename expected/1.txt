0xabcd
0xabcd

0xabcd
0x0abcd0

0xabcd
0xabcd0000

